### PR TITLE
[ticket/14089] Add posting_topic_title_after template event

### PR DESCRIPTION
**Pull request by Zoddo**
Allows to add some information after the topic title in the posting form

https://tracker.phpbb.com/browse/PHPBB3-14089

### CODE REVIEW

**Comment by Zoddo** [comment]
!set Event

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1052,6 +1052,14 @@ posting_poll_body_options_after
 * Since: 3.1.4-RC1
 * Purpose: Add content after the poll options on creating a poll
 
+posting_topic_title_append
```
Comment (marc1706) [inline phpBB/docs/events.md]: posting_topic_title_after

**Comment by marc1706** [comment]
This ticket was unfortunately also lost during the recent database outage. Please create another ticket and update the ticket id accordingly. Thank you.

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -1163,6 +1163,14 @@ posting_topic_title_before
 * Since: 3.1.6-RC1
 * Purpose: Allows to add some information on the left of the topic title in the posting form
 
+posting_topic_title_after
```
Comment (marc1706) [inline phpBB/docs/events.md]: This needs to be before "posting_topic_title_before"

**Comment by Zoddo** [comment]
!unset WIP

**Comment by Zoddo** [comment]
It's done :)

There is any chance to be merged in 3.1.6?

**Pull request merged**